src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -6,6 +6,20 @@ export PS1='#'
 unset PROMPT_COMMAND
 unset -f cd ls uname pwd date bc cat echo
 
+
```

Comment:
If the following is useful for debugging, should we enable it only when we debug?

Reply:
The overhead is near zero, so I'd rather leave it in instead of adding verbosity on the shell level (the bootstrapper does not look at the log level at all right now).